src/EnvelopedData.js

```diff
@@ -942,18 +990,15 @@ export default class EnvelopedData
 				const originator = new OriginatorIdentifierOrKey();
 				originator.variant = 3;
 				originator.value = new OriginatorPublicKey({ schema: asn1.result });
-				// There is option when we can stay with ECParameters, but here index prefer to avoid the params
-				if("algorithmParams" in originator.value.algorithm)
-					delete originator.value.algorithm.algorithmParams;

```

Comment:
This is what I was referring to in the description: I don't understand that comment and I can't think of any reason why we'd want to delete the `AlgorithmParams`. Any idea?